scss/partials/_cards.scss

```diff
@@ -72,7 +72,7 @@
 
     &-text {
       color: $cool-neutral-1;
-      font-size: 2.0rem;
+      font-size: 2rem;
 
```

Comment:
For some reason Prettier linter preffers `2` to `2.0`. 
That was not a change I made intentionally. 
Is there a specific reason to use `2.0` instead of `2`?

Reply:
There's no specific reason. 2rem is fine.